check_url.py

```diff
@@ -78,14 +78,15 @@ def check_ssh(hostname, user, private_key, command='hostname'):
     sys.exit(1)
 
 for line in csv_list.readlines():
+    print('\n')
```

Comment:
Why have a new line here. The goal of this part is to have a CSV file.

Reply:
Here, it's the output in the console. This add an empty line between each unit

As it on output we have somehting than 

```
http://github.com/ 😡 => https://github.com/ != 200


DNS entry not exist: http://jeijejieijejiwjaijaajeajajiajajajajia.com/ (HTTPConnectionPool(host
='jeijejieijejiwjaijaajeajajiajajajajia.com', port=80): Max retries exceeded with url: / (Cause
d by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd926ec7910>: Failed t
o establish a new connection: [Errno -2] Name or service not known')))
http://jeijejieijejiwjaijaajeajajiajajajajia.com/ 😡 => DNS entry not exist != 200


No TLS: https://toto.sogilis.com/ (HTTPSConnectionPool(host='toto.sogilis.com', port=443): Max
retries exceeded with url: / (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of p
rotocol (_ssl.c:1129)'))))
https://toto.sogilis.com/ 😡 => TLS Error != 200


Summary, list of errors:
```